### PR TITLE
Award Serializer for cassandra

### PR DESCRIPTION
**Pull request by ljeyaratnam**

### CODE REVIEW

**Comment by Spartakovic** [comment]
lgtm

File: atlas-cassandra/src/main/protobuf/common.proto [hunk 1]
```diff
@@ -150,4 +150,11 @@ message RelatedLink {
     optional string description = 6;
     optional string image = 7;
     optional string thumbnail = 8;
+}
+
+message Award {
+    optional string outcome = 1;
+    optional string title = 2;
+    optional string description = 3;
+    optional int32 year = 4;
```
Comment (Dalamar42) [inline atlas-cassandra/src/main/protobuf/common.proto]: Add a newline to the end of the file

File: atlas-core/src/main/java/org/atlasapi/entity/Award.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        Award copy = new Award();
+        copy.setOutcome(outcome);
+        copy.setTitle(title);
+        copy.setDescription(description);
+        copy.setYear(year);
+        return copy;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
```
Comment (Dalamar42) [inline atlas-core/src/main/java/org/atlasapi/entity/Award.java]: Same comment as the Owl equals/hashCode. This is wrong. Use IntelliJ to auto-gen it

**Comment by Dalamar42** [comment]
LGTM

**Pull request merged**